getWebpackConfig.js

```diff
@@ -38,6 +39,86 @@ function _getHtmlPlugin({ app, templatePath, isProduction }) {
   })
 }
 
+function _getPlugins({ apps, config, envVars, stats, defineVars, publicPaths, isProduction }) {
+  const { name: appTitle } = config
+
+  // Html Plugin: Generate one entry point HTML page per app
+  const htmlPlugins = apps.map((app) =>
+    _getHtmlPlugin({
+      app,
+      templatePath: config.templatePath,
+      isProduction,
+    }),
+  )
+  const plugins = [...htmlPlugins]
```

Comment:
```suggestion
  const plugins = htmlPlugins
```
you're not using any others